notification.go

```diff
@@ -0,0 +1,97 @@
+package ts3
+
+import (
+	"strings"
+)
+
+type notifyEvent string
```

Comment:
Any reason for the private type?

Reply:
Didn't see a reason why anyone would want this public but I'll make it public if it makes sense to you.

Reply:
Its not very nice to have a public method take a private type as an argument.

If a new category was added to TS, with a private type there would be no way for a user to support that without submitting a PR to add a new const. If its public ts3.NotifyCategory("newCategory") and they are away.

Reply:
Thanks, makes sense!